src/app/clusters/ota-requestor/OTARequestorStorageImpl.h

```diff
@@ -0,0 +1,52 @@
+/*
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include "OTARequestorStorage.h"
+
+namespace chip {
+
+class PersistentStorageDelegate;
+
+class OTARequestorStorageImpl : public OTARequestorStorage
```

Comment:
nit: This is a default, so it could be named as such

Reply:
The naming of default implementations is quite inconsistent and it's hard to make a decision based on the existing classes, but I can change that in the next PR.

Reply:
Renamed it already.